### ISSUE TITLE
Create web version of data model from the STIX 2.0 spec

### ISSUE THREAD

**Issue by gtback** [oasis-open/cti-documentation#19]
Similar conceptually to http://stixproject.github.io/data-model/

**Comment by SecurityDozo** [comment]
This information is at https://oasis-open.github.io/cti-documentation/stix/about.html on the bottom of the page. Its clarity is being fixed in the update.

**Comment by clenk** [comment]
Is this issue actually referring to the creation of separate pages for each object type, listing the properties and their data types, similar conceptually to (for example): http://stixproject.github.io/data-model/1.2/stix/STIXType/ or http://stixproject.github.io/data-model/1.2/report/ReportType/?

**Comment by gtback** [comment]
Yes, I was talking about individual pages for each data type, listing the properties, etc.
